Userland/Libraries/LibJS/Tests/builtins/RegExp/RegExp.prototype.exec.js

```diff
@@ -16,6 +16,7 @@ test("basic unnamed captures", () => {
     expect(res.length).toBe(2);
     expect(res[0]).toBe("fooooo");
     expect(res[1]).toBe("ooooo");
+    expect(RegExp.$1).toBe("ooooo");
```

Comment:
I think it's probably better to test this in its own separate test file? not too sure